engines/agos/midi.cpp

```diff
@@ -28,6 +28,8 @@
 #include "agos/agos.h"
 #include "agos/midi.h"
 
+#include "agos/drivers/accolade/adlib.cpp"
```

Comment:
This is is likely the cause of several builds now failing on buildbot due to multiple definitions of several symbols. All the symbols defined in `engines/agos/drivers/accolade/adlib.o` are also now also defined in `engines/agos/midi.o`.

Reply:
Damn. Mea Culpa. I should have spotted that. @Fantohin : Not the best idea as @criezy indicated. I can see why you did this though as the code structure is missing headers and does not make your changes easy.

Should be fixed by commit 3a0ed17e66e01d8b5175904ed821244be8b278b8 which I have pushed, but will see what buildbot indicates.